modules/tinymce/src/themes/silver/test/ts/browser/view/ViewButtonsTest.ts

```diff
@@ -126,13 +129,11 @@ describe('browser.tinymce.themes.silver.view.ViewButtonsTest', () => {
       assert.equal(getSvg(editor, 'button-without-toggle'), initialbuttonWithoutToggleButtonSvg, 'click should not toggle icon');
       clickViewButton(editor, 'button-without-toggle');
       assert.equal(getSvg(editor, 'button-without-toggle'), initialbuttonWithoutToggleButtonSvg, 'click should not toggle icon');
-
-      toggleView('myview1');
     });
 
     it('TINY-9616: if is active is true the button should have ViewButtonClasses.Ticked', async () => {
       const editor = hook.editor();
-      toggleView('myview2');
+      toggleView('myview1');
```

Comment:
isn't this covered by the: `afterEach(() => toggleView('myview1'));`?

Reply:
That line should be toggling the correct view, `myview1` instead of `myview2`, which is unregistered.